torch/_export/serde/serialize.py

```diff
@@ -1540,6 +1569,7 @@ def deserialize_sym_int(self, s: SymInt) -> Union[int, torch.SymInt]:
                                 compiler_max=vr.upper,  # type: ignore[arg-type]
                             )
 
+            sym = _translate_deserialized_sym_expr(sym)
```

Comment:
should this line go directly after line 1526?

Reply:
updated